defaultGzipBlacklist.js

```diff
@@ -0,0 +1,2 @@
+// By default, the following file types will not be gzipped
+module.exports = ['jpg', 'png', 'zip', 'gzip', 'png', 'xls', 'docx', 'gif'];
```

Comment:
xlsx. Classic xls is not a zip file. Others to add: xltx, dotx, pptx, ppsx, potx, sldx, gz.